GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/OptionSelect.cshtml

```diff
@@ -12,7 +12,7 @@
     <div class="options-container" id="@(Model.Id)" aria-label="@(Model.Label) filter" style="max-height: @(maxHeight)">
         <div class="js-auto-height-inner">
             <fieldset>
-                <legend>@(Model.Label)</legend>
+                <legend class="option-select-legend">@(Model.Label)</legend>

```

Comment:
we could use `class="govuk-visually-hidden"` instead

Reply:
That's way more sensible! Thanks 😊.